test/capture_web/router_test.exs

```diff
@@ -23,15 +23,17 @@ defmodule CaptureWeb.RouterTest do
         selected_answer: 5
       }
 
-      conn = conn(:post, "/responses", params)
-      |> CaptureWeb.Router.call(@opts)
+      conn =
+        conn(:post, "/responses", params)
+        |> CaptureWeb.Router.call(@opts)
 
       assert conn.status == 200
 
-      response = Response
-      |> Responses.for_survey(1)
-      |> Responses.for_question(1)
-      |> Repo.one
+      response =
```

Comment:
One way to tidy up these tests might be to create a function for finding a particular response row. I would suggest giving this a shot and see if you can do it!

Reply:
Sounds good - thanks!